docs/config/colormaps.rst

```diff
@@ -0,0 +1,130 @@
+.. _config_colormaps:
+
+Colormaps
+==========
+
+Each analysis task that includes plots with colormaps has a series of
+configuration options that can be used to manipulate the colormap.
+MPAS-Analysis supports two types of plots, each of which has an associated
+set of options for configuring its colormap.
+
+Supported Colormaps
+--------------------
+
+MPAS-Analysis supports all `matplotlib colormaps`_, all `cmocean colormaps`_,
+and five custom color maps: ``ferret``, ``erdc_iceFire_H``, ``erdc_iceFire_L``,
+``BuOr`` and ``Maximenko``.
+
+(To do: add an image of custom maps.)
+
+
+Indexed Colormaps
+------------------
+
+Indexed colormaps are used in association with contour plots in MPAS-Analysis.
+The following configuration options are associated with an indexed colormap::
+
+  # colormap name
+  colormapName = RdYlBu_r
+  # color indices into colormapName for filled contours
+  colormapIndices = [0, 40, 80, 110, 140, 170, 200, 230, 255]
+  # colormap levels/values for contour boundaries
+  colorbarLevels = [-2, 0, 2, 6, 10, 16, 22, 26, 28, 32]
+
+The indices are between 0 and 255, indicating the location within the colormap
+to sample.  The levels are the contour values to plot.  Since the region
+between two contours will be filled with a solid color specified by a colormap
+index, the number of levels is one greater than the number of indices.  The
+exception is when a separate shading should be used for values that fall
+outside the range of the contours, in which case the number of colormap indices
+is is one more than the number of levels values, and the first and last index
```

Comment:
Browsing through the documentation. Found this double 'is' above. Also, unrelated, but it may as well be fixed here: I remember a spelling error in observational_datasets: there is one 'Aquarius' that is spelled incorrectly.

Reply:
Both fixed, thanks!